src/request/export/Export.css

```diff
@@ -238,10 +232,6 @@ ul.nq-list li > .nq-icon {
     display: none;
 }
 
-.page#validate-words .validate-words .word-list {
-    margin: 0;
-}
-
 .page-footer {
     padding: 1rem;
 }
```

Comment:
The following rules about `.skip-words` need either to add `.skip`, or replaced by .skip everywhere, since `ValidateWords` does use `.skip`, which does not have a `:hover` rule defined. 

Potentially this can even go to `common.css`.

Reply:
Handled in #277.